manim/mobject/vector_field.py

```diff
@@ -594,7 +595,8 @@ def get_vector(self, point: np.ndarray):
             Additional arguments to be passed to the :class:`~.Vector` constructor
 
         """
-        output = np.array(self.func(point))
+        point = np.asarray(point)
```

Comment:
if i leave this out the docs compilation fails but i don't know why it suddenly passes in a list instead of an ndarray

Reply:
debugged on stream. :-)